sbncode/CAFMaker/CAFMaker_module.cc

```diff
@@ -1385,6 +1409,8 @@ void CAFMaker::produce(art::Event& evt) noexcept {
   StandardRecord* prec = &rec;
   fRecTree->SetBranchAddress("rec", &prec);
   fRecTree->Fill();
+  fBNBInfo.clear();
+  rec.hdr.pot = 0;
```

Comment:
Can you move these down two lines, below the srcol stuff? Otherwise you are wiping out the POT information from the version of the StandardRecord that gets saved into the art file. We don't use that for much/anything yet, but that will be an unpleasant surprise one day.